params/licenses.go

```diff
@@ -0,0 +1,37 @@
+package params
+
+import (
+	_ "embed"
+
+	"encoding/json"
+	"fmt"
+	"os"
+)
+
+type DepLicense struct {
+	Name    string  `json:"name"`
+	Version *string `json:"version"`
+	Type    string  `json:"type"`
+	Path    string  `json:"path"`
+}
+
+var (
+	// Polygon Edge License
+	License string
+
+	// Dependency Licenses
+	//go:embed bsd_licenses.json
+	bsdLicensesJSON string
+	BsdLicenses     []DepLicense
+)
+
+func init() {
+	if err := json.Unmarshal([]byte(bsdLicensesJSON), &BsdLicenses); err != nil {
+		fmt.Printf("failed to parse bsd_licenses.json")
+		os.Exit(1)
+	}
+}
```

Comment:
Do we need this to be an `init` method?
Why can't we unmarshal it programmatically manually?

Reply:
I'm afraid I don't get what `programmatically` means. What kind of code do you image you can write?

Reply:
Why do we need to rely on `init` to be present? Why can't we trigger this unmarshalling manually?

Reply:
I don't understand what manually means. But let it unmarshal ad-hoc

Reply:
I'll pretend we don't have this `init` method here, since we cleaned up `main` in [ac0a2e6](https://github.com/0xPolygon/polygon-edge/pull/407/commits/ac0a2e637a241829b9bda839bf60c20b1a9b0b1a) 🙂